Flutter/calendar/appointments.md

```diff
@@ -130,6 +130,129 @@ events.notifyListeners(CalendarDataSourceAction.reset, null);
 {% endhighlight %}
 {% endtabs %}
 
+## DataSource manipulation
+
+By call this method whenever the datasource changes, to notify the datasource may have. Need to notify the datasource action(add, remove and reset) through notify listener. In the following example, the appointment is added and notifyListeners is called in onPressed callback of the FloatingActionButton.
```

Comment:
In the following example, the appointment is added and notifyListeners is called in onPressed callback of the FloatingActionButton.

Not only for add action because below content for add, remove, reset ...

Reply:
I have checked and changed the content based on add, remove and reset. I have updated my changes in latest commit.